enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/SpatialIndexAcceptanceTest.scala

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.internal.cypher.acceptance
+
+import java.io.File
+import java.util.stream.Collectors
+
+import org.neo4j.cypher.GraphDatabaseTestSupport
+import org.neo4j.cypher.internal.javacompat.GraphDatabaseCypherService
+import org.neo4j.cypher.internal.util.v3_4.test_helpers.CypherFunSuite
+import org.neo4j.graphdb.config.Setting
+import org.neo4j.graphdb.factory.GraphDatabaseFactory
+import org.neo4j.graphdb.spatial.Point
+import org.neo4j.io.fs.FileUtils
+import org.neo4j.values.storable.{CoordinateReferenceSystem, PointValue, Values}
+
+import scala.collection.Map
+
+class SpatialIndexAcceptanceTest extends CypherFunSuite with GraphDatabaseTestSupport {
```

Comment:
You have changed all `createLabeledNode` followed by `MATCH ... SET ... RETURN` to `CREATE ... SET ... RETURN` in this file. I'm thinking that there may be a value of checking that setting points works both with `CREATE` and `MATCH` so maybe some tests should use one approach and some the other.